unittests/InstructionCountCI/AFP/VEX_map1.json

```diff
@@ -241,43 +250,47 @@
       ]
     },
     "vcvtsi2ss xmm0, xmm1, eax": {
-      "ExpectedInstructionCount": 2,
+      "ExpectedInstructionCount": 3,
       "Comment": [
         "Map 1 0b10 0x2A 128-bit"
       ],
       "ExpectedArm64ASM": [
         "mov v16.16b, v17.16b",
-        "scvtf s16, w4"
+        "scvtf s0, w4",
```

Comment:
Similar to `vsqrtss` from before, this looks like this should have worked with the previous version.
We first move xmm1 in to xmm0 at the start, then the `scvtf s16 (xmm0), w4 (eax)` should work because pseudocode:
```
constant boolean merge = IsMerging(FPCR);
fltval = if merge then V[d, fltsize] else Zeros(fltsize);
```
Using vd as the destination as the merge register (It's really confusing that instructions choose between vd, vn, and vm depending).

Same thought process for all the following vcvt* instructions after this.